res/css/views/rooms/_EventTile.scss

```diff
@@ -297,6 +297,8 @@ $left-gutter: 64px;
 
 /* De-zalgoing */
```

Comment:
Since this comment is only about the `overflow-y` part, please move it either above that line or inline to the right.